src/apiCalls.js

```diff
@@ -1,6 +1,11 @@
 const apiCalls = {
   getTopBirds: () => {
     return fetch('https://api.ebird.org/v2/data/obs/US-CO/recent?maxResults=30&key=jcsp15jhp6hi')
+  },
+
+  getSingleBird: (code) => {
+    return fetch(`https://api.ebird.org/v2/data/obs/US/recent/${code}?key=jcsp15jhp6hi`)
```

Comment:
This api call was added in case the generated random bird was not in either the `areaBirds` or the `sightings` list.